engine/battle/core.asm

```diff
@@ -2820,7 +2820,7 @@ SelectMenuItem_CursorDown:
 	jp SelectMenuItem
 
 Func_3d4f5:
-	bit 3, a
```

Comment:
This "3" is actually `BIT_START` in one of the 2 cases where this routine is called. But it's totally possible that it's just an accident of a bugged debug-only routine.

Reply:
Interesting! I've added an `ASSERT` for that.